KlimaDAO/klimadao
# Makka / 1333 - upgrade wallet connect to v2

## Description

Upgrade wallet connect to v2. 

**Note:** We need to create a projectId on https://cloud.walletconnect.com. I've temporarily created one and added it to enable testing but will remove once the official projectId is set up and added as an env variable.

Tests to cover:
- Login with wallets listed in the login modal
- Logout with all wallets and ensure user is logged out correctly.
- Switch accounts when logged in and ensure wallet switch works as expected.
- Login/Logout with a wallet in the wallet connect list and ensure it connects successfully with the correct address

## Related Ticket

Resolves #1333 

## Checklist

- [X] I have run `npm run build-all` without errors
- [X] I have formatted JS and TS files with `npm run format-all`
